### DISCUSSION TITLE
Recover from network failure

### DISCUSSION THREAD

**Discussion by swoutch** [QwikDev/qwik-evolution#168] [Proposals For Qwik]
### Which component is affected?

Qwik Runtime

### Describe the bug

# What I did:
1. `npm install`
2. `npm run preview`
3. In the chrome window that opens, I go to the developer tools, under the network tab, and set throttling to offline
4. I click on TODO demo 📝. Nothing happens
5. I then set throttling to No throttling
6. I click on TODO demo 📝 again. Nothing happens

# What I expect:
At step 6, I expect the todo demo to load and work fine.

# What actually happens:
At step 6, nothing happens.

### Reproduction

https://github.com/swoutch/qwik-recover-offline

### Steps to reproduce

1. `npm install`
2. `npm run preview`
3. In the chrome window that opens, go to the developer tools, under the network tab, and set throttling to offline
4. click on TODO demo 📝. Nothing happens
5. set throttling to No throttling
6. click on TODO demo 📝 again. Nothing happens

### System Info

```shell
System:
    OS: macOS 13.0.1
    CPU: (8) arm64 Apple M1
    Memory: 162.73 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 19.3.0 - ~/.nvm/versions/node/v19.3.0/bin/node
    Yarn: 1.22.18 - ~/.nvm/versions/node/v12.22.1/bin/yarn
    npm: 9.2.0 - ~/.nvm/versions/node/v19.3.0/bin/npm
  Browsers:
    Brave Browser: 110.1.48.164
    Chrome: 109.0.5414.119
    Firefox: 104.0.2
    Safari: 16.1
  npmPackages:
    @builder.io/qwik: 0.19.0 => 0.19.0 
    @builder.io/qwik-city: 0.3.0 => 0.3.0 
    undici: 5.20.0 => 5.20.0 
    vite: 4.1.2 => 4.1.2
```

### Additional Information

I'm not sure which qwik component is responsible for that, so I chose qwik runtime randomly

**Comment by gioboa** [comment]
This is an enhancement, no framework can manage this offline behaviour

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
